Pléiades is an optical satellite constellation providing very high-resolution products (50cm) with a 20km swath.
Both the space and the ground segment have been designed to provide information in record time, offering daily revisit capacity to any point on the globe and outstanding acquisition capabilities to meet the full spectrum of civil and military requirements.
The Pléiades constellation is the right answer to extract accurate information and bring powerful insight to make informed decisions.


 Product Description

50cm resolution products
20km swath
Complete ortho product range: Standard (automatic), Tailored and premium
12 bits dynamic range for high detection capabilities
Online tasking, purchasing and delivery


KEY FEATURES

    Very-high-resolution multispectral twin satellites [50-cm products]
    Daily revisit capacity and highly reactive tasking  
    Advanced agility including 5 collection scenarios: Target, Strip Mapping, Tri-Stereo, Corridor and Persistent Surveillance
    Truly available satellite resource capacity
    Stereo / Tristereo capacity and fresh archive




KEY BENEFITS

    Detailed & Precise: ideal for precision mapping and in-depth intelligence
    Fast & Frequent: particularly suitable for emergency response and up-todaily change detection
    Available & Reactive: with an industryleading collection success rate, Pléiades is the unanimous choice for new collections and timely decision-making
    Accurate orthorectified imagery as a standard & detailed and solid 3D models, affordable even in the most inaccessible locations




TASKING CAPABILITIES

    Joint optimized tasking together with Pléiades through “One Tasking” offer
    Online tasking through GeoStore, available 24/7, in full autonomy
    Perfect for small-area acquisitions with standard parameters or for urgent tasking
    Dedicated support and tracking from a satellite tasking expert. Ideal choice for complex AoIs or specific parameters
    Three satellite work plans a day ensuring easy handling of last-minute tasking requests
    All orbit contact allowing rapid access to critical information


TARGETED MARKETS & APPLICATIONS

    Defence
    Risk and disaster management
    Urban and mapping applications
    Civil Engineering






AIRBUS 
OneAtlas
Connecting Images from Space to Decisions on Earthx


https://www.intelligence-airbusds.com/oneatlas/




 Airbus Defence and Space intelligence account


OneAtlas is a unique collaborative environment to easily access premium imagery, perform large-scale image processing, extract industry specific insights and benefit from Airbus assets to develop your solutions.

The OneAtlas Services include:

    Living Library
    WorldDEM Streaming
    Verde

    Mobile
    Change Detection
    Starling


    Basemap
    Earth Monitor
    Ocean Finder
    Refinery Scanner



Airbus Defence and Space offers different options for purchasing products, services or solutions.

You can contact our Customer Service to assist you personally with your product or service request. Or you can use our range of web tools that allows you to order or purchase our satellite data and associated services online.


OneAtlas

OneAtlas provides access to premium imagery, large-scale image processing, industry-specific insights and more.



Data Services

OneAtlas provides users with easy access to multi-source, multi-resolution data and value-added layers that are immediately available off-the-shelf and easy to use.

Our Data service includes:

    Living Library
    Basemap
    WorldDEM Streaming service



Thematic Applications

OneAtlas combines premium data and industry-leading expertise to deliver Thematic Services. Designed to provide industry-specific insights enabling users to make better informed decisions.

Our Thematic Services include:

    Change Detection
    Ocean Finder
    Starling
    Verde
    RefineryScanner




Analytics

OneAtlas combines our data services with analytics capabilities that are suited for a variety of markets.

Our analytic capabilities include:

    Earth Monitor
    Change Detection



Up42

UP42 is an open platform and marketplace to build, run, and scale geospatial products.

It brings together high-resolution satellite data from OneAtlas, Sentinel Hub, sobloo and more, with ready-to-use algorithms — serving as a marketplace for the exchange and use of geospatial data and processing algorithms.

UP42 lowers the barrier of entry to use geospatial data by making it easy to access and analyze. Designed to enable users to extract insight for themselves, UP42 is bringing geospatial data within reach.




In [1]:
!apt-get install unzip

Reading package lists... Done
Building dependency tree       
Reading state information... Done
unzip is already the newest version (6.0-21ubuntu1).
0 upgraded, 0 newly installed, 0 to remove and 28 not upgraded.


In [2]:
!curl -o DS_PHR1A_201612271908229_FR1_PX_W123N37_0719_01350.zip "https://storage.googleapis.com/oneatlas-sandbox/USA-4/DS_PHR1A_201612271908229_FR1_PX_W123N37_0719_01350.zip?GoogleAccessId=storage-viewer@oneatlas-sandbox.iam.gserviceaccount.com&Expires=1577919599&Signature=LQy6s3CzwZ0p2KsGL9BABo15oQUtwalnbgzKp0ir%2FrGpW0Cai872VNIeWgk2natUbCIbnrjVBTJLYMLnsXU8gNeiOqG0eTb2wB%2FimeHTe0g9BwyYqNHrvdzEguDOqno0GNwJt2k43sJvy%2Bf7240RIa2Qm3NbEhnlY5yJTlXBI%2FkjlHbNYJCLqUBFh8i3l3byNsbb5cBs2BHJ9JT7EasjPtfIBzF4hccChfToEPUZkjNs0%2B%2FMqPS%2B3S%2FaxdH4dwujVrhQU607uuxbSRFZWaZU0eJVtfQo8WtuQFYaA40Kavp8rXb%2B5TTTvGESyJNBonlq21N5m8s%2Fh7qAOmlOChE6fQ%3D%3D"


  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 1111M  100 1111M    0     0   177M      0  0:00:06  0:00:06 --:--:--  187M


In [3]:
!curl -o DS_PHR1B_201401131909126_FR1_PX_W123N37_0717_02786.zip "https://storage.googleapis.com/oneatlas-sandbox/USA-4/DS_PHR1B_201401131909126_FR1_PX_W123N37_0717_02786.zip?GoogleAccessId=storage-viewer@oneatlas-sandbox.iam.gserviceaccount.com&Expires=1577919599&Signature=GdEquDfsuzG%2FyqgWq%2F8qPF3l84JhgVoNwkJJFflhedNYrRxCLXz%2B4MOrliZc0B3RIf6h4Rxf3NPk2IwfM4keeoFNAX7T8slYG1QAxuHUhWBkYjl7fGD0mTnQafkLeaMEvRu2K%2F%2BZeMJkrlYokjV2C4DNt1bVz4twkFxx5Km4ObGbO6I6YgRMyL9s855Y8PeEOrIXUGAvNXPM1amBIJ%2BnjXvCFT5zkHkRKiTpAURD3JkAIdnlOxPrNyHn2e43NjpvVvMdq1uAoK1RBPH%2F1X8q4OJV3pTWdRc%2FcFczOoqlEj7NLxF0J5hSna%2B4QeVIJlMVSbnB%2FCKOMUURL8bwWMJ70g%3D%3D"


  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 1637M  100 1637M    0     0  39.2M      0  0:00:41  0:00:41 --:--:-- 89.7M


In [4]:
!curl -o DS_PHR1B_201611181909035_FR1_PX_W123N37_0817_01200.zip "https://storage.googleapis.com/oneatlas-sandbox/USA-4/DS_PHR1B_201611181909035_FR1_PX_W123N37_0817_01200.zip?GoogleAccessId=storage-viewer@oneatlas-sandbox.iam.gserviceaccount.com&Expires=1577919599&Signature=GY027CjpF3KrprKEsXAS6Vv%2BMKG%2BUOYxnlgcY7p6Q%2BlHByuvqgekQcni9WUOiTF7dnDXShksX5s3okeX%2B7ZP7zf5dSZsDpoXgU44vQQBHl34yXiY4Jwfa4XEV87urApfuCv7zebhucVImOrApxw%2BhP%2FlaX%2BtTGRG%2FcDk3THB76SdNvoFY%2BcAGceoCBYvNIfok4SY0PJ%2FgSIdrGAbV8L7fzkxgmx6S7388v0wEM1unwxYz3BO7I50hb%2Fn3Iz76DJWbRy2rZSpVkMqKJZG9KUqpqJzaH23WXpvhg2PP7J7RLDPUNjiB%2Fzpq3SHNO%2BInwok37F03PiGzkB9cEQawYRDEw%3D%3D"


  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  935M  100  935M    0     0  32.5M      0  0:00:28  0:00:28 --:--:-- 55.1M


In [5]:
!curl -o DS_PHR1B_201702111904481_FR1_PX_W123N37_0719_01352.zip "https://storage.googleapis.com/oneatlas-sandbox/USA-4/DS_PHR1B_201702111904481_FR1_PX_W123N37_0719_01352.zip?GoogleAccessId=storage-viewer@oneatlas-sandbox.iam.gserviceaccount.com&Expires=1577919599&Signature=egz6PHGaiC3x02lYgl33f1K5mbbAoPwgDCS4LlBmQSr0KoI5b70AQsMevBwCn5L7Meeduypakdkco7OBITQxUpaHLTrhuT%2F6aA1lNuFdFmm3W4qeKbHJp2dCGGS6cs0KcdzkRNFdPapyjO90UIaJ7H32qn03hr3kxQxftYIMIFgYUuiP2iPlz7tBEKxlKKQIA56qJG7R95rSbolOqAq0awJ%2BakgrvRY64ExuQXG%2BTH3DhAJkZB4qU2mWB435N%2BDVkMNy0p%2Bg7bYpnWr6DngQgCvrVWcbWZHbcKhrzx5Bq96pxnme5VwNfsr9v4f08%2FDSvJvh8sJNr59RHTSYDx%2F0Hw%3D%3D"

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 1013M  100 1013M    0     0  37.8M      0  0:00:26  0:00:26 --:--:-- 26.5M


In [6]:
!ls

DS_PHR1A_201612271908229_FR1_PX_W123N37_0719_01350.zip
DS_PHR1B_201401131909126_FR1_PX_W123N37_0717_02786.zip
DS_PHR1B_201611181909035_FR1_PX_W123N37_0817_01200.zip
DS_PHR1B_201702111904481_FR1_PX_W123N37_0719_01352.zip
sample_data


In [7]:
!unzip DS_PHR1A_201612271908229_FR1_PX_W123N37_0719_01350.zip

Archive:  DS_PHR1A_201612271908229_FR1_PX_W123N37_0719_01350.zip
   creating: TPP1600634630/
 extracting: TPP1600634630/DELIVERY.PDF  
   creating: TPP1600634630/IMG_PHR1A_PMS_001/
 extracting: TPP1600634630/IMG_PHR1A_PMS_001/DIM_PHR1A_PMS_201612271908229_ORT_2326238101-001.XML  
 extracting: TPP1600634630/IMG_PHR1A_PMS_001/ICON_PHR1A_PMS_201612271908229_ORT_2326238101-001.JPG  
 extracting: TPP1600634630/IMG_PHR1A_PMS_001/IMG_PHR1A_PMS_201612271908229_ORT_2326238101-001_R1C1.J2W  
 extracting: TPP1600634630/IMG_PHR1A_PMS_001/IMG_PHR1A_PMS_201612271908229_ORT_2326238101-001_R1C1.JP2  
 extracting: TPP1600634630/IMG_PHR1A_PMS_001/INDEX.HTM  
   creating: TPP1600634630/IMG_PHR1A_PMS_001/LIBRARY/
 extracting: TPP1600634630/IMG_PHR1A_PMS_001/LIBRARY/LOGO.JPG  
 extracting: TPP1600634630/IMG_PHR1A_PMS_001/LIBRARY/STYLE.XSL  
   creating: TPP1600634630/IMG_PHR1A_PMS_001/LINEAGE/
 extracting: TPP1600634630/IMG_PHR1A_PMS_001/LINEAGE/GROUND_R3D_OR_SPOTView_N37W123_DIM.XML  
 extracting: TPP1600

In [8]:
!unzip DS_PHR1B_201401131909126_FR1_PX_W123N37_0717_02786.zip

Archive:  DS_PHR1B_201401131909126_FR1_PX_W123N37_0717_02786.zip
   creating: TPP1600634629/
 extracting: TPP1600634629/DELIVERY.PDF  
   creating: TPP1600634629/IMG_PHR1B_PMS_001/
 extracting: TPP1600634629/IMG_PHR1B_PMS_001/DIM_PHR1B_PMS_201401131909126_ORT_2326235101-001.XML  
 extracting: TPP1600634629/IMG_PHR1B_PMS_001/ICON_PHR1B_PMS_201401131909126_ORT_2326235101-001.JPG  
 extracting: TPP1600634629/IMG_PHR1B_PMS_001/IMG_PHR1B_PMS_201401131909126_ORT_2326235101-001_R1C1.J2W  
 extracting: TPP1600634629/IMG_PHR1B_PMS_001/IMG_PHR1B_PMS_201401131909126_ORT_2326235101-001_R1C1.JP2  
 extracting: TPP1600634629/IMG_PHR1B_PMS_001/INDEX.HTM  
   creating: TPP1600634629/IMG_PHR1B_PMS_001/LIBRARY/
 extracting: TPP1600634629/IMG_PHR1B_PMS_001/LIBRARY/LOGO.JPG  
 extracting: TPP1600634629/IMG_PHR1B_PMS_001/LIBRARY/STYLE.XSL  
   creating: TPP1600634629/IMG_PHR1B_PMS_001/LINEAGE/
 extracting: TPP1600634629/IMG_PHR1B_PMS_001/LINEAGE/HEIGHT_R3D_DT2_N37W123_DIM.XML  
 extracting: TPP1600634629/I

In [9]:
!unzip DS_PHR1B_201702111904481_FR1_PX_W123N37_0719_01352.zip

Archive:  DS_PHR1B_201702111904481_FR1_PX_W123N37_0719_01352.zip
   creating: TPP1600634627/
 extracting: TPP1600634627/DELIVERY.PDF  
   creating: TPP1600634627/IMG_PHR1B_PMS_001/
 extracting: TPP1600634627/IMG_PHR1B_PMS_001/DIM_PHR1B_PMS_201702111904481_ORT_2326232101-001.XML  
 extracting: TPP1600634627/IMG_PHR1B_PMS_001/ICON_PHR1B_PMS_201702111904481_ORT_2326232101-001.JPG  
 extracting: TPP1600634627/IMG_PHR1B_PMS_001/IMG_PHR1B_PMS_201702111904481_ORT_2326232101-001_R1C1.J2W  
 extracting: TPP1600634627/IMG_PHR1B_PMS_001/IMG_PHR1B_PMS_201702111904481_ORT_2326232101-001_R1C1.JP2  
 extracting: TPP1600634627/IMG_PHR1B_PMS_001/INDEX.HTM  
   creating: TPP1600634627/IMG_PHR1B_PMS_001/LIBRARY/
 extracting: TPP1600634627/IMG_PHR1B_PMS_001/LIBRARY/LOGO.JPG  
 extracting: TPP1600634627/IMG_PHR1B_PMS_001/LIBRARY/STYLE.XSL  
   creating: TPP1600634627/IMG_PHR1B_PMS_001/LINEAGE/
 extracting: TPP1600634627/IMG_PHR1B_PMS_001/LINEAGE/HEIGHT_R3D_DT2_N37W123_DIM.XML  
 extracting: TPP1600634627/I

In [10]:
!ls

DS_PHR1A_201612271908229_FR1_PX_W123N37_0719_01350.zip	sample_data
DS_PHR1B_201401131909126_FR1_PX_W123N37_0717_02786.zip	TPP1600634627
DS_PHR1B_201611181909035_FR1_PX_W123N37_0817_01200.zip	TPP1600634629
DS_PHR1B_201702111904481_FR1_PX_W123N37_0719_01352.zip	TPP1600634630


In [11]:
!wget https://sandbox.intelligence-airbusds.com/web/assets/files/Technical-Evaluation-Licence-December2014.pdf

--2019-10-30 14:44:08--  https://sandbox.intelligence-airbusds.com/web/assets/files/Technical-Evaluation-Licence-December2014.pdf
Resolving sandbox.intelligence-airbusds.com (sandbox.intelligence-airbusds.com)... 108.177.96.121, 2a00:1450:4013:c02::79
Connecting to sandbox.intelligence-airbusds.com (sandbox.intelligence-airbusds.com)|108.177.96.121|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/pdf]
Saving to: ‘Technical-Evaluation-Licence-December2014.pdf’

Technical-Evaluatio     [ <=>                ] 117.18K  --.-KB/s    in 0.02s   

2019-10-30 14:44:09 (5.40 MB/s) - ‘Technical-Evaluation-Licence-December2014.pdf’ saved [119991]



In [12]:
!pip install PyPDF2

     |████████████████████████████████| 81kB 5.5MB/s 
  Created wheel for PyPDF2: filename=PyPDF2-1.26.0-cp36-none-any.whl size=61085 sha256=dea5a8942c3874b768afa98d915a8e721810af5ce45e3c0fff9ebb24b2ff8317
  Stored in directory: /root/.cache/pip/wheels/53/84/19/35bc977c8bf5f0c23a8a011aa958acd4da4bbd7a229315c1b7
Successfully built PyPDF2


In [14]:
ls

DS_PHR1A_201612271908229_FR1_PX_W123N37_0719_01350.zip
DS_PHR1B_201401131909126_FR1_PX_W123N37_0717_02786.zip
DS_PHR1B_201611181909035_FR1_PX_W123N37_0817_01200.zip
DS_PHR1B_201702111904481_FR1_PX_W123N37_0719_01352.zip
sample_data/
Technical-Evaluation-Licence-December2014.pdf
TPP1600634627/
TPP1600634629/
TPP1600634630/


In [0]:
import PyPDF2
pdfFileObj = open('Technical-Evaluation-Licence-December2014.pdf', 'rb')
pdfReader = PyPDF2.PdfFileReader(pdfFileObj)

In [17]:
pageObj = pdfReader.getPage(0)
pageObj.extractText()

'Airbus Defence and Space\n \n \nOctober\n \n2014\n \n \n_____________________________________________________________________________________________________________________________\n__\n \nAirbus DS Geo SA\n \nSociété anonyme (325 089 589 R.C.S. Toulouse)\n \nwith share capital of 18\n \n022\n \n\n \nRegistered\n \noffice: 5 rue des Satellites \n\n \n31400 TOULOUSE \n\n \nFRANCE\n \nTVA : FR 30325089589 \n\n \nAPE/NAF : 7112B\n \n \nT\nECHNICAL \nE\nVALUATION \nL\nICENCE\n \n \nN\nON\n-\nE\nXCLUSIVE \nL\nICENCE \nT\nO \nU\nSE\n \nS\nTANDARD \nP\nRODUCT \nFOR \nT\nECHNICAL \nE\nVALUATION \nP\nURPOSE \nO\nNLY\n \n \nBy downloading\n \nand/or \nmanipulating\n \nand/or using \nthe PRODUCT, the \nEND\n-\nUSER \nsignifies that it\n \nunconditionally accepts \nall the terms and conditions of the present Technical Evaluation Licen\nc\ne Agreement (hereinafter referred to as the \n\n-\nUSER of this LICENCE is the condition upon which \nAIR\nBUS\n \nDS\n \nmakes the \nPRODUCT available to the 

Airbus Defence and Space October 2014

 

TECHNICAL EVALUATION LICENCE

NON-EXCLUSIVE LICENCE TO USE STANDARD PRODUCT FOR TECHNICAL EVALUATION
PURPOSE ONLY

By downloading and/or manipulating and/or using the PRODUCT, the END—USER signifies that it unconditionally accepts
all the terms and conditions of the present Technical Evaluation Licence Agreement (hereinafter referred to as the
“LICENCE"). The acceptance by the END—USER of this LICENCE is the condition upon which AIRBUS DS makes the
PRODUCT available to the END—USER. The END—USER of the PRODUCT sha|| guarantee that the PRODUCT and any
copy are used in strict compliance with the terms hereof.

This LICENCE is entered into by and between the END—USER and AIRBUS DS.

ARTICLE 1 — DEFINITIONS
“AIRBUS DS": means Airbus DS Geo SA.

“DERIVATIVE WORKS": means any derivative product or information developed by the END—USER from the PRODUCT,
which does not contain any imagery data from the PRODUCT and is irreversible and uncoupled from the source imagery
data of the PRODUCT. Notwithstanding the foregoing, by express exception, any Digital Elevation Model or Digital
Terrain Model (in any form whatsoever, i.e. database for instance) derived from a PRODUCT shall never be considered
as DERIVATIVE WORKS.

“END-USER": means either the person, acting in its own name, or the legal commercial business entity, including its
possible offices and branches in its country of residence, or the government agency, which is supplied with the
PRODUCT and accepts this LICENCE. When the PRODUCT is supplied to a government entity (civil agency, public
department, ...), the END—USER shall be deemed to be only such part of the government entity as located at the address
to which the PRODUCT is supplied, except upon AIRBUS DS‘ prior agreement in writing.

“EXTRACT": means an extract of a PRODUCT or VAP which may consist of:
0 an extract of 1024 X 1024 pixels maximum; or
o if supplied with the PRODUCT (preview file) by AIRBUS DS, the sub—sample (ratio 32 toward original PAN (8
toward original XS) ) of the original image of the PRODUCT.

“PRODUCT": means any standard satellite imagery and/or radar product that is provided to the END—USER for technical
evaluation purpose.

“VAP": means any product developed by the END—USER, which contains imagery data from the PRODUCT, and
resulting in a significant modification of the PRODUCT, through technical manipulations and/or addition of other data.

Notwithstanding the foregoing, by express exception, any Digital Elevation Model or Digital Terrain Model derived from a
PRODUCT shall always be considered as a VAP.

ARTICLE 2 - LICENCE
2.1 Permitted Uses

The END—USER is hereby granted by AIRBUS DS a limited, non—exclusive, non—transferable, licence:

(a) to make an unlimited number of copies of the PRODUCT for the Permitted Uses specified in this Article 2.1;

(b) to install the PRODUCT on as many individual computers as needed in its premises, including internal computer

network (with the express exclusion of the Internet) for the Permitted Uses specified in this Article 2.1;

( ) to use the PRODUCT for internal technical evaluation purposes only;

(d) to alter or modify the PRODUCT to produce VAP and/or DERIVATIVE WORKS for internal technical evaluation
purposes only;

(e) to use any VAP and/or DERIVATIVE WORKS for internal technical evaluation purposes only;

(f) to use a PRODUCT, VAP and/or DERIVATIVE WORKS in the framework of competitive analysis (such as
benchmarking) for internal technical evaluation purposes only;

(9) to make the PRODUCT and/or any VAP available to contractors and consultants, only for technical evaluation on
behalf of the END—USER, subject to such contractors and consultants agreeing in writing, in advance, (I) to be bound
by the same limitations on use as applicable to the END—USER, and (II) to return the PRODUCT to END—USER, and
to keep no copy thereof, upon completion of the contracting or consulting engagement. By express exception,
Elevation PRODUCTS (3D) and TerraSAR—X PRODUCTS shall not be made available and/or transferred to any third
party without AIRBUS DS‘ prior written consent; and

 

Airbus DS Geo SA

Société anonyme (325 089 589 R.C.S. Toulouse) with share capital of 18 022 949 €
Registered office: 5 rue des Satellites — 31400 TOULOUSE — FRANCE

TVA : FR 30325089589 — APEINAF : 71123

Airbus Defence and Space October 2014

 

(h) to print one EXTRACT of the PRODUCT for internal technical evaluation purposes only, with the credit of the
corresponding PRODUCT mentioned in article 3.4 conspicuously displayed and written in full.

All rights not expressly granted by AIRBUS DS under the present Article 2.1 are hereby retained by AIRBUS DS.

2.2 Prohibited Uses

The END—USER shall not, and shall guarantee that any contractor or consultant engaged as per the provisions of Article

2.1 (9) does not:

(a) use the PRODUCT, VAP and/or DERIVATIVE WORKS for any use other than internal technical evaluation,
especially, any commercial use;

b) alter or remove any copyright notice or proprietary legend contained in or on the PRODUCT;

c reproduce, transmit, disseminate, or make available in any way the PRODUCT and/or any product created from the
PRODUCT, in whole or in part, to any third party other than a contractor or consultant engaged as per the provisions
of Article 2.1 (g), by any means, free of charge or in return for payment;

(d) publish or communicate in any way a competitive analysis (such as benchmarking) of any PRODUCT, VAP and/or

DERIVATIVE WORKS except article 2.1 f);
(e) transfer the PRODUCT outside of the territory of establishment of the END—USER; or
(f) do anything not expressly permitted under Article 2.1.

AA
V

2.3 Technical Evaluation Results
The END—USER shall inform AIRBUS D8 of the results of the evaluation it has performed.
ARTICLE 3 — INTELLECTUAL PROPERTY RIGHTS

3.1 The data contained in the PRODUCT are the property of:

0 Centre National d‘Etudes Spatiales (CNES) for SPOT 1 to 5 and PLEIADES sate||ites imagery data, SPOTMaps
and SPOT DEM data;

AIRBUS DS for SPOT 6 and 7 sate||ites imagery data (© AIRBUS DS);

National Space Organization (NSPO) for FORMOSAT—Z satellite imagery data;

DEIMOS Imaging for DEIMOS—1 satellite imagery data and DEIMOS Maps data;

Federal Republic of Germany for TerraSAR—X satellite data which are distributed by Infoterra GmbH; and

AIRBUS DS and IGN France, in common, for Reference3D and SPOT DEM Precision data.

3.2 The PRODUCT is protected by French and international copyright laws.

3.3 In addition, the PRODUCT and the satellite imagery data contained therein are protected by articles L 341—1 to 343—7
of the French Code of Intellectual Property Law as amended by the statute of 1 July 1998, relative to database
copyright and to similar statutes in European countries that have incorporated EU Directive n° 96/9 of 11 March 1996
on database copyright into their |aws.

3.4 The PRODUCT, VAP and/or EXTRACT, when displayed and/or printed in accordance with the Permitted Uses
specified in Article 2.1 shall include the AIRBUS DS logo and the following credits conspicuously displayed:

0 For SPOT 1 to 5, PLEIADES satellite imagery data, SPOTMaps and SPOT DEM data: “© CNES (year of
acquisition), Distribution AIRBUS DS"

0 For SPOT 6 and 7 imagery data: “© AIRBUS DS (year of acquisition)"

0 For FORMOSAT—Z satellite imagery data: “© NSPO (year of production), Distribution AstriumServices/Spot
Image SA"

0 For DEIMOS—1 satellite imagery data and DEIMOS Maps data: “© Deimos Imaging (year of production),
Distribution AIRBUS DS"

0 For TerraSAR—X satellite data: “© DLR e. V. (year of acquisition), Distribution AIRBUS DS/Infoterra GmbH"

0 For Reference3D and SPOT DEM Precision data: “© CNES (year of reception of the data)"; and “© AIRBUS
DS (year of production of the PRODUCT)" and “© IGN France (year of production of the PRODUCT)"

ARTICLE 4 — WARRANTY — LIABILITY

4.1 AIRBUS DS warrants that it has sufficient ownership rights in the PRODUCT to make the PRODUCT available to the
END—USER under the terms hereof.

4.2 The PRODUCT is complex; AIRBUS DS does not warrant that the PRODUCT is free of bugs, errors, defects or
omissions, and that operation of the PRODUCT will be error—free or uninterrupted nor that all non—conformities will or
can be corrected. It does not warrant that the PRODUCT shall meet the END—USER‘s requirements or expectations,
or shall be fit for the END—USER‘s intended purposes. There are no express or implied warranties of fitness or
merchantability given in connection with the use of the PRODUCT or any other warranty. AIRBUS DS disclaims all
other warranties or liabilities not expressly provided in Articles 4.1 and 4.2.

 

Airbus DS Geo SA

Société anonyme (325 089 589 R.C.S. Toulouse) with share capital of 18 022 949 €
Registered office: 5 rue des Satellites — 31400 TOULOUSE — FRANCE

TVA : FR 30325089589 — APEINAF : 7112B

Airbus Defence and Space October 2014

 

4.3 In no event shall AIRBUS DS, nor its licensors, nor anybody having contributed to development and/or production

and/or delivery of the PRODUCT, be liable for any claim, damage or loss incurred by the END—USER, including
without limitation indirect, compensatory, consequential, incidental, special, incorporeal or exemplary damages
arising out of the use of or inability to use the PRODUCT, and shall not be subject to legal action in this respect.

The financial cumulative liability of AIRBUS DS and its licensors and of anybody having contributed to the
development and/or production and/or delivery of the PRODUCT shall in any case be limited to five hundred euros
(500€).

ARTICLE 5 — MISCELLANEOUS

5.1

This LICENCE shall run for six (6) months from the date the PRODUCT is made available to the END USER. It is
granted free of royalties.

5.2 AIRBUS DS may, in addition to all other remedies to which it may be entitled under this LICENCE or at law,

terminate this LICENCE by notice in writing if the END—USER breaches any provision hereof. The END—USER shall
have no claim to any kind of indemnity in this case. Upon termination or expiration of this LICENCE, the END—USER
shall return to AIRBUS DS the PRODUCT and VAP and all copies thereof.

5.3 The END—USER shall not transfer part or all of this LICENCE.
5.4 In the event that any provision of this LICENCE is declared invalid or unenforceable, the remaining provisions hereof

shall be applicable.

5.5 This LICENCE is governed by the laws of France. All disputes shall be referred to the courts of Toulouse, France.

 

(Date)

 

(Name of the organization of the END USER)

 

(Name of the representative of the END USER)

 

(Title of the representative of the END USER)

 

(Signature of the representative of the END USER)

 

Airbus DS Geo SA

Société anonyme (325 089 589 R.C.S. Toulouse) with share capital of 18 022 949 €
Registered office: 5 rue des Satellites — 31400 TOULOUSE — FRANCE

TVA : FR 30325089589 — APEINAF : 7112B
